tests/rpc/rpc_03-chain_test.go

```diff
@@ -135,3 +137,162 @@ func TestChainRPC(t *testing.T) {
 	errList := utils.TearDown(t, nodes)
 	require.Len(t, errList, 0)
 }
+
+func TestChainSubscriptionRPC(t *testing.T) {
+	if utils.MODE != rpcSuite {
+		_, _ = fmt.Fprintln(os.Stdout, "Going to skip RPC suite tests")
+		return
+	}
+
+	testCases := []*testCase{
+		{
+			description: "test chain_subscribeNewHeads",
+			method:      "chain_subscribeNewHeads",
+			expected: []interface{}{1,
+				map[string](interface{}){
+					"subscription": float64(1),
+					"result": map[string](interface{}){
+						"number":         "0x04",
```

Comment:
is there a reason the expected block is block 4?

Reply:
No, 1 makes more sense, I've updated this.